src/main/java/net/glowstone/block/ItemTable.java

```diff
@@ -391,7 +395,6 @@ private void registerBuiltins() {
                 .YELLOW_GLAZED_TERRACOTTA));
         reg(Material.CHORUS_FLOWER, new BlockChorusFlower());
         reg(Material.CHORUS_PLANT, new BlockChorusPlant());
-
```

Comment:
Don't remove the separation here between blocks and items.

Reply:
Reverted and added comments.